notebooks/source/bayesian_hierarchical_linear_regression.ipynb

```diff
@@ -0,0 +1,677 @@
GNGZaJDSK1fJY9FjSteSQN3XW+yHjSZNa4HJtMoOsVJEijeeFEEV0+DEdoYJj9sMzYefsMcqdfoxlhayTD1BOnQ605sLyjo+IWr+THuuHkmYUE1XKSMdDy7YM+ScO5IkWFkmsWY2KJdJ10RNHicixqYjd3gFy4TxT5SHW9Gd5cqpKNnmMjaMUtW13kTnncpQ6ShSOOC6OnKvq2DR0jVbMEhZKFjKMUNJARR5KJlG09kUYtdbq3NFunHbkyYskmd+5hypzeZSWQuj6cqQu7UX4CX05Am+5ESk/pkcqnGMicU9qIYaIWUwoLvC042o6q3ZAww2JpSLTPPo/Ba19vSshGYkFp/oN1ngNOBIljGKkYaDRqre0f6d/mGza3J9MIoTGtcHx3x27dk+7HpEOo3H7JhGCuarDSHeaXMrgd0vRIzsE8+gNFUvF4WKL2NtxjC0kWXX8vT5TcQgjiaYipFJULRcn1ljZc9TWSdshdT9iyQkII8na4jRhlMAcaWf3KJajckt1bzkl1ODE6HK6EWDrCnqg6gQcLtlcsLoHSxM4QtF7eCfPhZME7Q8YrjvD9My38LwF+novZ+XKN6Fpz9/4/f8Ct43fxpe2f4k/3/TnDKQHuGXXLWxZ3MIXX/JFMmbm95/geUIpxfb7Zqj+6tuclT2NOz/1NC/9k9NZf8HgCcdKqfj4nU/z3hevWyZnAOvXncqn1/07f3eHxq/P3EA6oeM7IXd9aQfDYymuKL8dLZUlrs8wNfoYgxevwt56IwtTL+e692xk6JwNzI+/kgd/fhuh/BBfKyVIZU9HFzqn957Ob2s+TednvObhtZjJs3nryn7CWPLpe/fxbxs1Tk8JAs1h/MwreO9khGf8J+Z5d3NeczVe+Wqmzd8wO7iCOzfbiMDg34LPI/0cu+qK6zbexeiPbsbXdO4wK1w3+D5uXnkt6+bnufQXX+csOc55g10UhM2W8jBP3LebVySX+NLQKxjr2Mfdg/+dTYd/yWTvHm7+0H9j+62LrLi3iHfq/ex48k5eUJ7l8Q9eTLK7i8kntjD1bz9iafcUnqbRW+7kXx6Z4HuvOu+EuV654XSu/JO3c89X/4m3/OOXWTzocN+te+hflUPT4NGfTnLVn53B2nOeH/Hx9rdESdLnDDzncUZXkr6bT6f0nT0kVnRgDp9ogP0uPnP3Pt77knUM5o6n9l0Zk7+86lT+5dcHThK0k/i/AgUIL0KTgFJE5SoyX0SSBtIgFV7zmdP74kadsFxl72mrODfQWsaykth+jC5cioZkSvMYa7uoqkIidcG6SKCilopZEhAowjBqWQlCoJkmyrWRTRvl+1RDIKVYaUVkMgH73BNTrH4dumDEXOz4RM0AvbAXM7UOS3VRKlZRrndcvVOtXCf2FQmg0zuMlRxFtc2KZ1IsG4glB5sesZZopSxaNlq9AV2jrfNpilysiGkZJ/v2TiGbNvRVAROp4GDBIpyfpzy6AtuHsUKBaGCQXXM1xlImAxWfnfoxcuayJaKQnxonqQ/jp49apeqY2t+8D0mlEAoKBvTJluLjBdkWMQglKD9gttjADWMaXoQIfZTZijhZXsSBJYsz3AjDFETG8WRk56EFzlu/4rjPDhWaDKLRaYUkQ8lit8dCzUU/xhrVii5CKtTvKEsd+ezJYoOikPTQ+n9CF3ihpOYEy2lfh4pNxhOSEDg/0JGqlTIpw4hjR5kIYqp2gNdwSUpw4hbVrWoK5UYslZt0uweopU4lGUjMMCJKwu4GHOjUkGUHVmSQjoMyBNu37CEjSwxlhzhFLlEZPg8lnp2kDVR8ip06GAaxHSEDCWlIyCYNJ6QSBNSdkBXOARKsIHZ8DhSOn5en5+vs1RtkY0XeA6Pk4A+noV1brdpRi2cSdwljRdUO6KOlzFhuBmTbDhfHDRmoBRR7EhixQvNTbG/m6bG6kS7kG8fYP+2t/2Szhp9NMNR2gHjlKkhJ6FYwrHFm7fMpxv10pU1W9bZ+fyxJnSw0SRkaQ/kJNK0MuQtp5kvIXBd6V/dyxLTDidFk69qPheWFTC61UhUPG5IIbbmwqb8W0BlvIdW1Cq/nwtb1W01EOoXzO1x/52wNEhAIyFXm0GRxmaCBYmdCMhwLVsTiuDRZKUH4DZLSISBB5RgnQrkZsL3SpAvQECzqrUkbjY+cVeC2U6nPH+tBHuPucGQDvR4R9h8tiQCWz/9wpUGo4OJkGtkWuYli1SJnMkRXPkrB4bKDGTfpSK5q1QvKmMi2WcilULFBDkEpBGQ7VROFhlgmYlpSJ/YjYquB7Ewi9QRRtYrR03pmaCpAxdrxzei9kG3ZJOUEXMJz42QN2h8oarVtbH3ijwHBwMDVLC7dwY6db0PK598s7/82Hpp7iC8+9UW+cfU3uGnjTbxs9cv47iu+i6mZ/O3Df0v8e8K1zxdKKn7z3X3sfXiBF543w9DLXssr33sOD3x//wkFrQD37y9QbPq840Vrjz+PUlzykusYi6b48n1Po5Tige+P0zWQ5krrPWjNRdSqi9l91ZWMcxY/rW/inotvZ2nkdm6/ZZK5yQX2anfzg3U/4/aa4E97Jef0nU5Pqoc1XWspamvpVmV2xxtpzn2W19z5Wq6/+37uMwJu2PkANzy5hQ/8cjPfftzjO2tcHtwwxAP6NRQWMsyOX0r18Uu4YGGRRGTyg/gVXJ/+GH/dcxNrzce5b+95+At7ubMyi+lluSdxLl85fYzVv7ob39ZZcd56ToseYq5hsmGxSW+jSdSc5/zGk3xv71t4y+6v8vcrb2DWnGXz9s1c+pZ17O3v59GVGzh/fj/++pVs+/YtfO+jf8nkhz+C2L6HcjpFl97J6w4+wCs//w/86JHJE+Ya4OyXXkvv0BD3feGT/Opfn+KlF03ymlcV+KN3r+Oat5/Jr2/dc1xz2OdCc/MCHZeOIJ6HrGVybRe5F49SuW389yo7bpuqMFls8meXrX3G799w4SiHSzZPTD13DvhJnMTvQywVU0WbOIpASsx
```

Comment:
Thanks for mentioning (my name is Du Phan). I think ending the tutorial with `Finally, I hope ... predictions.` is better for the tutorial.